src/sql/platform/capabilities/common/capabilitiesService.ts

```diff
@@ -28,6 +28,7 @@ export interface ConnectionProviderProperties {
 	azureResource?: string;
 	connectionOptions: azdata.ConnectionOption[];
 	isQueryProvider?: boolean;
+	supportedExecutionPlanFileExtensions?: string[];
```

Comment:
seems a little "ad-hoc" to put this here.  I think the intended design would be there would be different ProviderProperties for different systems, like connection, query, notebooks, etc.  And those ProviderProperties objects would be bundled in the ProviderFeatures object (or something like that).  I don't think we need to refactor as part of this PR, but would be good to improve later.